### ISSUE TITLE
Trouble configuring the layout to print properly

### ISSUE THREAD

**Issue by MaggieMcGee** [ExLibrisGroup/spineomatic-cloudapp#19]
Hi Josh,
I am using the Demco 1 1/2 x 7/8 layout. I've tried configuring the Demco layout with the dimension of the label. Sadly, the label dimensions do no translate and work as input.  I was able to adjust the height and width of the label. I am having trouble with where the labels being printing. I thought it would be the left margin. I have the left margin set to 0 and it starts too far over. I've tired other values without success. How can I get the labels to begin printing in the correct position. Please see attached.
![label](https://user-images.githubusercontent.com/81188041/115776711-6c1f5200-a382-11eb-9311-907eb6264979.jpg)

The settings I have are attached.
![spineOmatic_Layout_Setttings](https://user-images.githubusercontent.com/81188041/115776292-f4512780-a381-11eb-8ca4-a45aeba1dea5.jpg)

Thanks,
Maggie

**Comment by mck1777** [comment]
Maggie
I ran into this problem. My firefox will not let me clear the margins at work yet at home it works fine. Our IT department is trying to figure out the reason [lot of suggestions, no answer yet]. I am able to clear the margins in chrome at work no problem.

To work in chrome this is how I accomplished it
Click on more settings
![image](https://user-images.githubusercontent.com/72558867/115777876-d4bafe80-a383-11eb-8274-583d166bd0b4.png)

I had to set margins to none and scale to custom and 100
![image](https://user-images.githubusercontent.com/72558867/115777982-f5835400-a383-11eb-81b1-d8b915b84a73.png)

I hope this helps.

Maryclare
